fearless/Modules/Staking/StakingBondMore/StakingBondMoreViewController.swift

```diff
@@ -43,7 +43,6 @@ final class StakingBondMoreViewController: UIViewController, ViewHolder {
     }
 
     private func setupAmountInputView() {
-        rootView.amountInputView.textField.keyboardType = .decimalPad
```

Comment:
why?

Reply:
It always decimal pad and configured inside view